src/SFA.DAS.AssessorService.Web/Views/Shared/_Layout.cshtml

```diff
@@ -43,7 +43,7 @@
                                     <a href="/" class="das-user-navigation__link">Settings</a>
                                     <ul role="menu" class="das-user-navigation__sub-menu js-hidden" aria-expanded="false">
                                         <li class="das-user-navigation__sub-menu-list-item" role="menuitem">
-                                            <a href="/" class="das-user-navigation__sub-menu-link">Change your sign-in details</a>
+                                            <a href="@Url.Action("ChangeSignInDetails", "Account")" class="das-user-navigation__sub-menu-link">Change your sign-in details</a>
```

Comment:
I think this needs to be behind a toggle, or the action needs to decide which view to show

Reply:
Toggle already in place line no 40. Please check